cluster/saltbase/salt/top.sls

```diff
@@ -72,6 +72,7 @@ base:
     - docker
     - kubelet
 {% endif %}
+    - kube-proxy
```

Comment:
Minor nit: would this be better on line 44 before all the optional configuration?  (In that case, perhaps lines 53-56 could move up too).  Not sure if I think about these configs like the original authors do, but my argument is that it's easier to discover the defaults if they're not intermixed with the optional features.